ci/ci.sh

```diff
@@ -123,7 +123,7 @@ case $HOST_TARGET in
     # Some targets are only partially supported.
     MIRI_TEST_TARGET=x86_64-unknown-freebsd run_tests_minimal hello integer vec panic/panic concurrency/simple pthread-threadname libc-getentropy libc-getrandom libc-misc libc-fs atomic env align num_cpus
     MIRI_TEST_TARGET=i686-unknown-freebsd run_tests_minimal hello integer vec panic/panic concurrency/simple pthread-threadname libc-getentropy libc-getrandom libc-misc libc-fs atomic env align num_cpus
-
+    MIRI_TEST_TARGET=x86_64-unknown-illumos run_tests_minimal no_std
```

Comment:
This won't test most of the code you added. Please make sure we don't have dead code.

Since this is not a no_std target, please definitely add `hello`.

For the threading functions, you probably should add `concurrency/simple concurrency/thread_name pthread-threadname`.

For errno and since you edited that test, `libc-misc` should probably also be added. However libc-misc has a bunch of stuff so I'm not sure if it will fully work.

For getrandom/getentropy, add `libc-getrandom libc-getentropy`.